apps/workbench/config/application.yml.example

```diff
@@ -24,6 +24,9 @@ production:
   secret_token: ~
 
   # You probably also want to point to your API server.
-  arvados_login_base: https://arvados.local:3030/login
-  arvados_v1_base: https://arvados.local:3030/arvados/v1
+  arvados_login_base: https://uuid_prefix.your.domain/login
+  arvados_v1_base: https://uuid_prefix.your.domain/arvados/v1
   arvados_insecure_https: false
+
+  # You might also want to name your workbench
+  site_name: ~
```

Comment:
I would simply leave the hint as "option_name: ~" so it won't effect your decision, unless the option name itself changed. However I understand your concerning so I withdrew my changes on this.
Maybe we could add the following line at the end of every install doc
> For more advanced configs, please refer to ```application.default.yml``` in ```/var/www/arvados-xxx/config```.